Unity-Technologies/Graphics
# Quality audit - Specify the way that Volume Gizmos are being rendered

---
### Purpose of this PR
- Adding user preferences to specify how the Volume Gizmos are being rendered
- Adding a system to extend Core Render Pipeliene Settings Panel
- Unify Gizmo rendering with `Volume`and `CustomPassVolume`

![gizmo_visibility](https://user-images.githubusercontent.com/76204843/127350656-670e2324-b89f-487b-886f-71f364fb1a0a.gif)

---
### Testing status
- Check gizmos are correctly changed
- Check the preferences are applied, saved and restored
